proxy/src/main/java/io/openlineage/proxy/service/ProxyService.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.openlineage.proxy.service;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Lists;
+import io.openlineage.proxy.ProxyConfig;
+import io.openlineage.proxy.api.models.ConsoleLineageStream;
+import io.openlineage.proxy.api.models.KafkaLineageStream;
+import io.openlineage.proxy.api.models.LineageEvent;
+import io.openlineage.proxy.api.models.LineageStream;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import lombok.NonNull;
+
+public final class ProxyService
+{
+  private static final Logger log = LoggerFactory.getLogger(ProxyService.class);
+
+  private final ImmutableSet<LineageStream> lineageStreams;
+
+
+  /**
+   * Constructor reviews the content of the config file and sets up the appropriate lineage streams.
+   *
+   * @param config configuration properties supplied to the application
+   */
+  public ProxyService(final ProxyConfig config)
+  {
+    /*
+     * There are two supported lineage streams.  The first is logging to the console.  This is activated by default or if requested in the config.
```

Comment:
This can be simplified if we define a `LineageStreamsFactory`, see [earlier](https://github.com/OpenLineage/OpenLineage/pull/400/files#r764421224) comment